app/controllers/v1/events_controller.rb

```diff
@@ -9,20 +9,13 @@ def show
   end
 
   def create
-    @event = Event.new(event_params)
-    if @event.save
-      render json: @event
-    else
-      render 'validation_error', status: :forbidden
-    end
+    @event = Event.create!(event_params)
+    render json: @event
   end
 
   def update
-    if @event.update_attributes(event_params)
-      render json: @event
-    else
-      render 'validation_error', status: :forbidden
-    end
+    @event.update_attributes!(event_params)
```

Comment:
>you should use update_attributes , if you have a model, like this:
book.update_attributes(params[:book])
and you should use update, when you want on the criteria, like this:
Book.where(id: params[:id]).update(params[:book])

https://groups.google.com/forum/#!topic/mongoid/c_bAhwtHWHs
I didn't know about it. Good question!!